package.json

```diff
@@ -59,24 +59,30 @@
     "@commitlint/cli": "^7.5.2",
     "@commitlint/config-angular": "^7.5.0",
     "@commitlint/prompt-cli": "^7.5.0",
+    "@edx/eslint-config": "^1.1.6",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^24.5.0",
     "babel-loader": "^8.0.6",
     "coveralls": "^3.0.3",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.11.2",
-    "eslint-config-edx": "^4.0.4",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react-hooks": "^1.7.0",
```

Comment:
Why is this needed?

Reply:
I followed the directions here: https://github.com/edx/eslint-config
(Namely `npx install-peerdeps --dev @edx/eslint-config`), and this is everything it installed.